src/08_comprehensions.py

```diff
@@ -39,8 +32,5 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
-for num in x:
-    if (int(num) % 2) == 0:
-        y.append(num)
+y = [ num for num in x if(int(num) % 2) == 0 ]
```

Comment:
Good work here! Note, in Python we actually don't need the parens for the conditional => `y = [num for num in x if int(i) % 2 == 0]` will suffice.